patches/tModLoader/Terraria/ModLoader/UI/UILoadMods.cs

```diff
@@ -36,25 +36,27 @@ public void SetLoadStage(string stageText, int modCount = -1)
 	{
 		this.stageText = stageText;
 		this.modCount = modCount;
-		if (modCount < 0) SetProgressText(Language.GetTextValue(stageText));
+		if (modCount < 0) SetProgressText(Language.GetTextValue(stageText), Language.GetTextValue(stageText));
 		Progress = 0;
 		SubProgressText = "";
 	}
 
-	private void SetProgressText(string text, string logText = null)
+	// uitext might have chat tags, text is the same without chat tags, logText might be unique
+	private void SetProgressText(string uitext, string text, string logText = null)
```

Comment:
Probably better to do `text, uiText = null, logText = null`

or just strip the chat tags here.